### ISSUE TITLE
Loss functions for classification - logits/probabilities (page 472)

### ISSUE THREAD

**Issue by labdmitriy** [rasbt/machine-learning-book#29]
Hi Sebastian,
 
There is the same value on the picture on the page 472 for ``y_pred`` in the columns for probabilities (BCELoss) and logits (BCEWithLogitsLoss): 0.8
Probably the value for the first column (BCELoss) is 0.69, which is equal to sigmoid(0.8)?
 
Thank you.

**Comment by labdmitriy** [comment]
Probably y_pred=0.8 for BCEWithLogitsLoss is more consistent with the code you provided in the book (and the second row is already consistent with the code):
```
logits = torch.tensor([0.8])
probas = torch.sigmoid(logits)
```

**Comment by rasbt** [comment]
Good point

**Comment by haydenliu** [comment]
Thanks Dmitry and Seb.
Merge request: https://github.com/rasbt/machine-learning-book/pull/33

**Comment by rasbt** [comment]
Thanks, just merged!
